ethers-solc/Cargo.toml

```diff
@@ -26,10 +26,14 @@ md-5 = "0.9.1"
 thiserror = "1.0.30"
 hex = "0.4.3"
 colored = "2.0.0"
+home = "0.5.3"
+svm = { package = "svm-rs", git = "https://github.com/roynalnaruto/svm-rs", optional = true }
 
 [dev-dependencies]
 tokio = { version = "1.12.0", features = ["full"] }
 tempdir = "0.3.7"
 
 [features]
 async = ["tokio", "futures-util"]
+svm-rs = ["svm"]
```

Comment:
Think a feature is created automatically for any optional dependency you have, so I wonder if you can just remove this extra feature and use svm?